mk/README.md

```diff
@@ -2,3 +2,4 @@ This directory contains the _Common_ makefiles for the UVM and Core testbenches
 - Cloning third party repositories (Core RTL, libraries, ISGs, RISC-V compliance library, etc.)
 - Invoking the toolchain to compile directed and/or random test programs into machine code for simulation
 - Invoking the toolchain to compile compliance test programs for simulation
+- Invoking the toolchain to import the source code in DVT Eclipse IDE
```

Comment:
I'm not sure this is correct.   In core-v-verif (and indeed, within most of the RISC-V  community), the term "toolchain" is typicalled used to mean the cross-compiler used to compile/assemble either C or RISC-V assembler source code into RISC-V machine code.   I doubt that we need to use the toolchain to import DVT.